### PR TITLE
Added .toISO method for ISO 8601 output

### PR DESCRIPTION
**Pull request by chamini2**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,6 +35,38 @@ PostgresInterval.prototype.toPostgres = function () {
...
+  days: 'D',
+  hours: 'H',
+  minutes: 'M',
+  seconds: 'S'
+}
+var dateProperties = ['years', 'months', 'days']
+var timeProperties = ['hours', 'minutes', 'seconds']
+// according to ISO 8601
+PostgresInterval.prototype.toISO = function () {
+  var buildProperty = function (property) {
```
Comment (bendrucker) [inline index.js]: Named functions please, pop it at the bottom (below the return)

**Review by bendrucker — CHANGES_REQUESTED** [review]
One small comment, otherwise good stuff!

File: index.js [hunk 2]
```diff
@@ -47,7 +47,14 @@ var dateProperties = ['years', 'months', 'days']
 var timeProperties = ['hours', 'minutes', 'seconds']
 // according to ISO 8601
 PostgresInterval.prototype.toISO = function () {
-  var buildProperty = function (property) {
+  var datePart = dateProperties
```
Comment (bendrucker) [inline index.js]: Can you also add line breaks between the two vars, the return, and the function declaration?

**Review by bendrucker — CHANGES_REQUESTED** [review]
One more minor thing. Please also add docs to the readme.

File: readme.md [hunk 3]
```diff
@@ -26,7 +28,7 @@ interval.toPostgres()
 
 ##### pgInterval
 
-*Required*  
+*Required*
```
Comment (bendrucker) [inline readme.md]: Please make sure this trailing whitespace stays, it's important to proper formatting

**Review by bendrucker — CHANGES_REQUESTED** [review]

**Comment by chamini2** [comment]
All done, @bendrucker!

**Comment by bendrucker** [comment]
Thank you!

**Pull request merged**